environment/src/main/java/com/sequenceiq/environment/environment/dto/EnvironmentLoadBalancerDto.java

```diff
@@ -0,0 +1,122 @@
+package com.sequenceiq.environment.environment.dto;
+
+import java.util.Set;
+
+import com.sequenceiq.cloudbreak.common.event.Payload;
+import com.sequenceiq.common.api.type.PublicEndpointAccessGateway;
+import com.sequenceiq.environment.environment.domain.Environment;
+
+public class EnvironmentLoadBalancerDto implements Payload {
```

Comment:
**question:** It looks to me like this `Dto` is being used as a value object that is passed between Java/Spring services.

I'm used to `Dto`s as wire-level objects at some point -- will the DTO be used that way in the future?

This also assume that DTO actually stands for Data Transfer Object.

Reply:
I've changed this object so that it no longer has the `Environment` member. I think that was the only thing that was potentially non serializable, and we have way to look it up later in the part of the code that actually needs it.